File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        count=0
        length = len(nums)
        for i in range(0,length-1):
            for j in range(i+1,length):
                if nums[i]+nums[j]==target:
                    count=1
                    return [i,j]
        if count==0:
            return [-1 ]


        

File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        map <int,int> mpp;
        vector <int> ans;
        for(int i=0;i<nums.size();i++){
            if(mpp.find(target-nums[i])!=mpp.end()){
                return {i,mpp[target-nums[i]]};
            }else{
                mpp[nums[i]]=i;
            }
        }
        return {-1,-1};
    }
};